static/scss/profile.scss

```diff
@@ -18,19 +18,38 @@
 
   .actions {
     justify-content: flex-end;
+    margin-top: 0;
   }
 
   .profile-image-container {
     margin-right: 40px;
   }
 
-  .name-and-headline {
-    .name {
-      font-size: 24px;
-    }
+  .basic-info {
+    justify-content: flex-start;
+
+    .name-headline-location {
+      .name {
+        font-size: 24px;
+        color: $navy;
+        font-weight: bold;
+      }
+
+      .headline {
+        font-size: 16px;
+        color: $font-grey;
+      }
+
+      .location {
+        font-size: 16px;
+        margin-top: 22px;
```

Comment:
I'm guessing the first image in the issue is the desired look (it's not indicated)? If so,I don't think this should have some/all of this top margin, the mockups show the leading between name, headline, and location to be of similar size, but I'm seeing this:
![Screenshot_2019-06-05 Profile for Nathan Levesque MIT Open Learning](https://user-images.githubusercontent.com/28598/58969385-9d3e0b80-8785-11e9-9550-195088dd9354.png)

Reply:
ah, the issue was that my user's profile doesn't have the location set, so I didn't see this. I'll remove the `margin-top` on that.